ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -25,17 +25,19 @@ FDISK_CMDS
   fi
 
   echo "Mounting SD CARD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
-  mount /dev/mmcblk0p1 /media/scratch | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
-  cat /etc/rc.local.logs >> /media/scratch/sdmount.log
+  mount /dev/mmcblk0p1 /media/plugin-data | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+  cat /etc/rc.local.logs >> /media/plugin-data/sdmount.log
 
 else
   echo "Error: SD CARD not found will continue without SD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
 fi
 
-#Removing PI user and updating ROOT Password
-userdel -r -f pi | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
-echo "'pi' user removed" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
-
 #Installing all packages from /isodebs/
 dpkg -i /isodebs/*
 echo "Packages In /isodebs/ Installed" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+
+mkdir -p /media/plugin-data/k3s/kubelet
+mkdir -p /media/plugin-data/k3s/rancher
+ln -s /media/plugin-data/k3s/kubelet/ /var/lib/kubelet
+ln -s /media/plugin-data/k3s/rancher/ /var/lib/rancher
+INSTALL_K3S_SKIP_DOWNLOAD=true /etc/installK3S.sh
```

Comment:
Please add a log to `/etc/rc.local.logs` at the end of this script indicating that things were successful

Reply:
will do